services/cron/tasks.go

```diff
@@ -147,6 +147,11 @@ func GetTask(name string) *Task {
 func RegisterTask(name string, config Config, fun func(context.Context, *user_model.User, Config) error) error {
 	log.Debug("Registering task: %s", name)
 
+	i18nKey := "admin.dashboard." + name
+	if value := translation.NewLocale("en-US").Tr(i18nKey); value == i18nKey {
+		return fmt.Errorf("translation is missing for task %q, please add translation for %q", name, i18nKey)
+	}
+
```

Comment:
Doesn't this need a isProd check too?

Reply:
This should only happen for developers, https://github.com/go-gitea/gitea/pull/18646#discussion_r800307598